resources/sdk/purecloudpython/templates/README.mustache

```diff
@@ -111,6 +111,82 @@ PureCloudPlatformClientV2.configuration.proxy_password = 'YOUR_PROXY_PASSWORD'
+
+`PureCloudPlatformClientV2.logger.LogLevel` values:
+1. LTrace (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers, Response Headers)
+2. LDebug (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers)
+3. LError (HTTP Method, URL, Request Body, Response Body, HTTP Status Code, Request Headers, Response Headers)
+4. LNone - default
+
+`PureCloudPlatformClientV2.logger.LogFormat` values:
+1. JSON
+2. TEXT - default
+
+By default, the request and response bodies are not logged because these can contain PII. Be mindful of this data if choosing to log it.  
+To log to a file, provide a file path to the the `PureCloudPlatformClientV2.configuration.logger.log_file_path` property. SDK users are responsible for the rotation of the log file.
+
+Example logging configuration:
+```python
+PureCloudPlatformClientV2.configuration.logger.log_level = PureCloudPlatformClientV2.logger.LogLevel.LError
+PureCloudPlatformClientV2.configuration.logger.log_request_body = True
+PureCloudPlatformClientV2.configuration.logger.log_response_body = True
+PureCloudPlatformClientV2.configuration.logger.log_format = PureCloudPlatformClientV2.logger.LogFormat.TEXT
+PureCloudPlatformClientV2.configuration.logger.log_to_console = False
+PureCloudPlatformClientV2.configuration.logger.log_file_path = "/var/log/pythonsdk.log"
+```
+
+#### Configuration file
+
+A number of configuration parameters can be applied using a configuration file. There are two sources for this file:
+
+1. The SDK will look for `%HOME%\.genesyscloudpython\config` or `%USERPROFILE%\.genesyscloudpython\config` if `%HOME%` is not set on Windows, or `$HOME/.genesyscloudpython/config` on Unix.
+2. Set the `PureCloudPlatformClientV2.configuration.config_file_path` property to the path of your configuration file.
+
+The SDK will check every 30 seconds to see if the config file has been updated, regardless of whether a config file was present at start-up. To disable this behaviour, set `PureCloudPlatformClientV2.configuration.live_reload_config` to false.  
+INI and JSON formats are supported. See below for examples of configuration values in both formats:
+
+INI:
+```ini
+[logging]
+log_level = trace
+log_format = text
+log_to_console = false
+log_file_path = /var/log/pythonsdk.log
+log_response_body = false
+log_request_body = false
+[reauthentication]
+refresh_access_token = true
+refresh_token_wait_max = 10
+[general]
+live_reload_config = true
+host = https://api.mypurecloud.com
+```
+
+JSON:
+```json
+{
+    "logging": {
+        "log_level": "trace",
+        "log_format": "text",
+        "log_to_console": false,
+        "log_file_path": "/var/log/pythonsdk.log",
+        "log_response_body": false,
+        "log_request_body": false
+    },
+    "reauthentication": {
+        "refresh_access_token": true,
+        "refresh_token_wait_max": 10
+    },
+    "general": {
+        "live_reload_config": true,
+        "host": "https://api.mypurecloud.com"
+    }
+}
+```
+
```

Comment:
Do we need a mechanism to show how the trace levels would be set programmatically via the config object.  I always thought the file was the first option and then programmatic config is the second option.

Reply:
Yeah, this code block shows how all the logging config can be set programmatically: https://github.com/purecloudlabs/platform-client-sdk-common/pull/157/files#diff-58afaa4a115cb1ef50340943bb96aa5fc4e2907723a2ecda2bb71d1c7d4eab86R133